src/containers/Client.tsx

```diff
@@ -55,75 +77,122 @@ export const Client = ({
+        videoPerks: { enabled: false, simulcast: undefined, encodings: undefined },
+      },
+    ]);
+  };
 
   return (
-    <div className="card w-150 bg-base-100 shadow-xl m-2 indicator">
-      <CloseButton
-        onClick={() => {
-          remove(roomId);
-          setTimeout(() => {
-            refetchIfNeeded();
-          }, 500);
-        }}
-      />
-      <div className="card-body m-2">
-        <h1 className="card-title">
-          Client: <span className="text-xs">{peerId}</span>
-          <CopyToClipboardButton text={peerId} />{" "}
-        </h1>
-        <BadgeStatus status={fullState?.status} />
-        <div>
-          <div className="flex flex-row items-center">
-            Token:
-            {token && (
-              <button className="btn btn-sm m-2 btn-error" onClick={removeToken}>
-                <svg
-                  xmlns="http://www.w3.org/2000/svg"
-                  className="h-6 w-6"
-                  fill="none"
-                  viewBox="0 0 24 24"
-                  stroke="currentColor"
-                >
-                  <path strokeLinecap="round" strokeLinejoin="round" strokeWidth="2" d="M6 18L18 6M6 6l12 12" />
-                </svg>
-              </button>
-            )}
-          </div>
-          {token ? (
-            <div>
-              <span className="break-words text-xs">{token}</span>
-              <CopyToClipboardButton text={token} />
-            </div>
-          ) : (
-            <div>
-              <input
-                type="text"
-                placeholder="Type here"
-                className="input input-bordered w-full max-w-xs"
-                onChange={(e) => {
-                  setTokenInput(e.target.value);
-                }}
-              />
-              <button className="btn btn-sm m-2 btn-success" onClick={() => setToken(tokenInput)}>
-                Save token
-              </button>
-            </div>
-          )}
-        </div>
+    <div className="flex flex-col">
+      <div className="card w-150 bg-base-100 shadow-xl m-2 indicator">
+        <CloseButton
+          onClick={() => {
+            remove(roomId);
+            setTimeout(() => {
+              refetchIfNeeded();
+            }, 500);
+          }}
+        />
+        <div className="card-body">
+          <div className="flex flex-row">
+            <h1 className="card-title z-10 relative">
+              Client: <span className="text-xs">{peerId}</span>
+              <div
+                className="tooltip tooltip-top tooltip-primary absolute -ml-3 -mt-1 -z-20 "
+                data-tip={fullState?.status}
+              >
+                <BadgeStatus status={fullState?.status} />
+              </div>
+              <CopyToClipboardButton text={peerId} />{" "}
+            </h1>
 
-        <div className="flex flex-row justify-between">
-          <div className="flex flex-row flex-wrap items-start content-start">
             {disconnect ? (
```

Comment:
Could you also change this lines to this? Because it fix the button state when user changes room tab. 

```
{fullState.status === "joined" ? (
              <button
                className="btn btn-sm btn-error m-2"
                onClick={() => {
                  disconnect?.();
```